[🐛 Bug]: Hub fails to start with --log-level DEBUG

### Description

**Environment:**
- Selenium version: 4.33.0
- Java version: OpenJDK 17
- OS: macOS (Sonoma)

**Steps to reproduce:**
- `java -jar selenium-server-4.33.0.jar hub --log-level DEBUG`

**Expected Result:**
- Hub starts and logs appear at DEBUG level.

**Actual Result:**
- No output is shown.
- Hub process silently fails (no logs, no errors).
- Confirmed that `--log /dev/stdout` and `-verbose` show class loading but no actual boot logic.

**Workarounds**
- `--log-level INFO` works
- `--log-level FINE` works

I've tried older versions up to 4.15, none of them work with `--log-level DEBUG`, however there's a log message appearing with 4.15 - the log does not appear with any other version between 4.33 and 4.15:
```
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.openqa.selenium.grid.Bootstrap.runMain(Bootstrap.java:77)
	at org.openqa.selenium.grid.Bootstrap.main(Bootstrap.java:70)
Caused by: org.openqa.selenium.grid.config.ConfigException: Unable to determine log level from DEBUG
	at org.openqa.selenium.grid.log.LoggingOptions.setLoggingLevel(LoggingOptions.java:79)
	at org.openqa.selenium.grid.log.LoggingOptions.configureLogging(LoggingOptions.java:115)
	at org.openqa.selenium.grid.TemplateGridCommand.lambda$configure$4(TemplateGridCommand.java:120)
	at org.openqa.selenium.grid.Main.launch(Main.java:83)
	at org.openqa.selenium.grid.Main.go(Main.java:56)
	at org.openqa.selenium.grid.Main.main(Main.java:41)
	... 6 more
```

### Reproducible Code

```shell
java -jar selenium-server-4.33.0.jar hub --log-level DEBUG
```

### Debugging Logs

```logs
No logs at all.
```

--- a/java/src/org/openqa/selenium/grid/log/LoggingOptions.java
+++ b/java/src/org/openqa/selenium/grid/log/LoggingOptions.java
@@ -76,7 +76,9 @@ public class LoggingOptions {
     try {
       level = Level.parse(configLevel.toUpperCase(Locale.ROOT));
     } catch (IllegalArgumentException e) {
-      throw new ConfigException("Unable to determine log level from " + configLevel);
+      // Logger is not configured yet
+      new ConfigException("Unable to determine log level from " + configLevel +
+       ". Using default " + DEFAULT_LOG_LEVEL).printStackTrace();
     }
   }
 

--- a/java/src/org/openqa/selenium/grid/log/LoggingOptions.java
+++ b/java/src/org/openqa/selenium/grid/log/LoggingOptions.java
@@ -77,8 +77,14 @@ public class LoggingOptions {
       level = Level.parse(configLevel.toUpperCase(Locale.ROOT));
     } catch (IllegalArgumentException e) {
       // Logger is not configured yet
-      new ConfigException("Unable to determine log level from " + configLevel +
-       ". Using default " + DEFAULT_LOG_LEVEL).printStackTrace();
+      new ConfigException(
+              "Unable to determine log level from "
+                  + configLevel
+                  + ". Using default "
+                  + DEFAULT_LOG_LEVEL
+                  + ". Available log levels can be found at"
+                  + " https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Level.html")
+          .printStackTrace();
     }
   }
 

--- a/java/src/org/openqa/selenium/grid/log/LoggingOptions.java
+++ b/java/src/org/openqa/selenium/grid/log/LoggingOptions.java
@@ -24,6 +24,7 @@ import java.io.UncheckedIOException;
 import java.io.UnsupportedEncodingException;
 import java.util.Arrays;
 import java.util.Enumeration;
+import java.util.List;
 import java.util.Locale;
 import java.util.logging.Handler;
 import java.util.logging.Level;
@@ -49,6 +50,17 @@ public class LoggingOptions {
   private final Config config;
   private Level level = Level.INFO;
   public static final String DEFAULT_LOG_TIMESTAMP_FORMAT = "HH:mm:ss.SSS";
+  private static final List<Level> DEFAULT_LOG_LEVELS =
+      Arrays.asList(
+          Level.ALL,
+          Level.INFO,
+          Level.CONFIG,
+          Level.FINE,
+          Level.FINER,
+          Level.FINEST,
+          Level.OFF,
+          Level.SEVERE,
+          Level.WARNING);
 
   public LoggingOptions(Config config) {
     this.config = Require.nonNull("Config", config);
@@ -82,8 +94,8 @@ public class LoggingOptions {
                   + configLevel
                   + ". Using default "
                   + DEFAULT_LOG_LEVEL
-                  + ". Available log levels can be found at"
-                  + " https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Level.html")
+                  + ". Available log levels are: "
+                  + DEFAULT_LOG_LEVELS)
           .printStackTrace();
     }
   }

